src/kernel-install/90-loaderentry.install

```diff
@@ -77,14 +77,10 @@ else
     LOADER_ENTRY="$BOOT_ROOT/loader/entries/$MACHINE_ID-$KERNEL_VERSION.conf"
 fi
 
-if ! [ -d "$ENTRY_DIR_ABS" ]; then
-    if [ "$KERNEL_INSTALL_VERBOSE" -gt 0 ]; then
-        echo "+mkdir -v -p $ENTRY_DIR_ABS"
-        mkdir -v -p "$ENTRY_DIR_ABS"
-    else
-        mkdir -p "$ENTRY_DIR_ABS"
-    fi
-fi
+[ -d "$ENTRY_DIR_ABS" ] || {
+    echo "Error: entry directory '$ENTRY_DIR_ABS' does not exist" >&2
+    exit 1
+}
```

Comment:
I don't mind switching to this style of ifs but let's do it in a separate commit and all at once instead of using two different styles?

Reply:
I backed out the style change. It doesn't make the code shorter, so we might just as well keep the old style.